pkg/instancetype/instancetype.go

```diff
@@ -811,39 +811,41 @@ Volume -> DataVolumeSource -> DataVolumeTemplate -> DataVolumeSourcePVC -> Persi
 Volume -> DataVolumeSource -> DataVolumeTemplate -> DataVolumeSourceRef -> DataSource
 Volume -> DataVolumeSource -> DataVolumeTemplate -> DataVolumeSourceRef -> DataSource -> PersistentVolumeClaim
 */
-func (m *InstancetypeMethods) inferDefaultsFromVolumes(vm *virtv1.VirtualMachine, inferFromVolumeName, defaultNameLabel, defaultKindLabel string) (defaultName, defaultKind string, err error) {
+func (m *InstancetypeMethods) inferDefaultsFromVolumes(vm *virtv1.VirtualMachine, inferFromVolumeName, defaultNameLabel, defaultKindLabel string) (string, string, error) {
 	for _, volume := range vm.Spec.Template.Spec.Volumes {
 		if volume.Name != inferFromVolumeName {
 			continue
 		}
 		if volume.PersistentVolumeClaim != nil {
-			return m.inferDefaultsFromPVC(volume.PersistentVolumeClaim.ClaimName, vm.Namespace, defaultNameLabel, defaultKindLabel)
+			defaultName, defaultKind, err := m.inferDefaultsFromPVC(volume.PersistentVolumeClaim.ClaimName, vm.Namespace, defaultNameLabel, defaultKindLabel)
+			return defaultName, defaultKind, err
```

Comment:
Why do we need to change this? Doing just `return m.inferDefaultsFromPVC(...)` should be fine here.

Reply:
I thought it improves readability of the code